pkg/controllers/data.go

```diff
@@ -0,0 +1,147 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/interactor"
+	"github.com/gorilla/mux"
+)
+
+type DataController struct {
+	DataInteractor interactor.DataInteractor
+}
+
+type DataInterface interface {
+	GetAll()
+	GetByID()
+	Create()
+}
```

Comment:
Keep an eye out for [function grouping and ordering](https://docs.google.com/document/d/1w61-0sp_QkuaCI3qepc9yxh0AEkCBuS0bOyKSahsOkQ/edit#heading=h.fwxbbk14050i) :stuck_out_tongue: 

Interfaces should come first :smile:

Reply:
Done!